### PR TITLE
Fix the bug that function `register_handler` was not working properly

### PR DESCRIPTION
**Pull request by d-hayashi**
## What?
Fix function `register_handler` in `pydtk.db.v4.handlers`

## Why?
Bug fix

## Screenshot or video [Optional]
before:
```python
>>> from pydtk.db import DBHandler
Failed to load handlers in meta.py
>>> 

```

after:
```python
>>> from pydtk.db import DBHandler
>>> 

```

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/105#" title="Author self-approved">d-hayashi</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pydtk/db/v4/handlers/__init__.py
Comment (yadach) [inline pydtk/db/v4/handlers/__init__.py:44]:
```suggestion
                os.path.join(
                    "pydtk.db.v4.handlers", str(os.path.splitext(filename)[0])
                ).replace(os.sep, ".")
```

**Review by yadach — COMMENTED** [review]
How about something like this?
I'm not sure if suggestion is working well.

File: pydtk/db/v4/handlers/__init__.py
Comment (d-hayashi) [reply pydtk/db/v4/handlers/__init__.py]: It worked!
Thank you 👍

**Review by yadach — APPROVED** [review]
LGTM

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/105#" title="Author self-approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/pydtk/pull/105#pullrequestreview-829773200" title="Approved">yadach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi,yadach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
